ui/lib/sync/addon/routes/secrets/destinations/destination/edit.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Route from '@ember/routing/route';
+import { service } from '@ember/service';
+
+import type Store from '@ember-data/store';
+
+type RouteParams = {
+  name: string;
+  type: string;
+};
+
+// originally this route was inheriting the model (Ember Data destination model) from the destination parent route
```

Comment:
Is it not possible to do `this.modelFor()` and grab the `destination` from the return of the model hook in `destination.ts` ?

Reply:
Eventually yes and that will be updated in the next PR that addresses the create and edit routes. Right now the destination route is pulling the data from the API client and this route still needs the Ember Data model. This won't last long though I'll have that PR coming up shortly.

Reply:
Oh doh! thank you